### ISSUE TITLE
Paragraphs functionality

### ISSUE THREAD

**Issue by juliaford** [thinkshout/bene#70]
## Overview
Use paragraphs module to allow building of long form content. 3 Paragraph types: Rich Text; Full-width block; tile block (up to 3 tiles)
## This ticket is done when:
- [ ] item
- [ ] item

**Comment by unclegcb** [comment]
@tauno assigned to you for implementation review. Should go to design/strategy for naming advice next.

**Comment by tauno** [comment]
@unclegcb Plain text for Featured and Tile text fields? I could see some simple formatting as necessary - bold, italic, paragraphs and maybe links - that does imply a additional input format/wysiwyg config though. Also, the homepage design shows a 3 column paragraph with images above the title/text which would either require inline images or special handling to make the background image an inline image.

**Comment by unclegcb** [comment]
Yeah I wavered on that: figured making it rich text is an easy future improvement since the data will be compatible: you just want a more restricted wysiwyg than the general one, which is a minor hassle. Happy to change if you lean that way. ("Background image" is just a copy/paste error: will fix.)

**Comment by tauno** [comment]
I moved the WYSIWYG/input format config to the base setup task (#51). I also updated the spec here to use the restricted input format for the featured and tile text areas.

@juliaford and @vickibrown want to weigh in on the naming of these paragraph types? Rich text, Featured and Tiles right now. 

Also will need image dimensions from @vickibrown

**Comment by unclegcb** [comment]
@vickibrown and PM's like:

Field is called "Content"
Paragraph types are called "Rich Text"; "Feature"; "Tiles"

**Comment by vickibrown** [comment]
@unclegcb We could also consider, "Full-Width Feature"

**Comment by tauno** [comment]
Another consideration - and the reason we've gone down the "CTA" naming route on other projects is to use the naming to convey the purpose of the thing. "Call to Action" or "Full-Width Call to Action" helps tell the user what they should use that thing for. This type of naming is less useful for the three column blocky-paraphraphy-chunky-thing.

**Comment by unclegcb** [comment]
Awkward here, though, because those full-width paragraphs can also be used as content interludes without a button: so no action called out.

**Comment by tauno** [comment]
Good point.

**Comment by unclegcb** [comment]
Perhaps that's a UX problem, though, and we should separate into 2 options for UX reasons. I collapsed them in the architecture for development efficiency, really.

**Comment by unclegcb** [comment]
rich_text: "Rich Text"
full_width_feature: "Feature"
tiles: "Tiles"
media_item: "Media"

**Comment by komejo** [comment]
Ready for review.
